rllib/agents/dqn/apex.py

```diff
@@ -84,6 +84,32 @@ def update_target_based_on_num_steps_trained(trainer, fetches):
         trainer.state["num_target_updates"] += 1
 
 
+# Update worker weights as they finish generating experiences.
+class UpdateWorkerWeights:
+    def __init__(self, learner_thread, workers, max_weight_sync_delay):
+        self.learner_thread = learner_thread
+        self.workers = workers
+        self.steps_since_update = collections.defaultdict(int)
+        self.max_weight_sync_delay = max_weight_sync_delay
+        self.weights = None
+
+    def __call__(self, item):
```

Comment:
nit/suggestion: What if you type hint these args? One question I have when reviewing is what are these args, and how do I know where to look if I wanted to add arguments/parameters?